src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/AvoidOutParameters.cs

```diff
@@ -58,7 +58,7 @@ public void Analyze(SymbolAnalysisContext analysisContext)
                 var methodSymbol = (IMethodSymbol)analysisContext.Symbol;
                 if (methodSymbol.IsOverride ||
                     !analysisContext.Options.MatchesConfiguredVisibility(Rule, methodSymbol, analysisContext.Compilation) ||
-                    methodSymbol.IsImplementationOfAnyImplicitInterfaceMember())
+                    methodSymbol.IsImplementationOfAnyInterfaceMember())
```

Comment:
NOTE: This change doesn't change any behavior. The test passes with and without it due to https://github.com/dotnet/roslyn-analyzers/issues/5878. I'm changing it just to match the intent.